tests/e2e/vLLM/configs/sparse_24_qwen.yaml

```diff
@@ -0,0 +1,8 @@
+cadence: "nightly"
+test_type: "regression"
+model: Qwen/Qwen2.5-0.5B
+recipe: tests/e2e/vLLM/recipes/Sparse_2of4/recipe_sparse_2of4.yaml
+scheme: sparse2of4_only
```

Comment:
let's do sparse2of4_fp8 -- sparse2of4 is currently broken in vLLM right now (bandaid fix atm)

Reply:
moved to `sparse2of4_fp8_dynamic_qwen.yaml`, hitting an error during tests on A100s that it has insufficient compute capability (has 80, needs 90)

Reply:
after some more digging, looks like this is expected on A100s, only hopper and above have compute capability 9.0.